example/src/components/Footer.tsx

```diff
@@ -0,0 +1,89 @@
+import { BookOpenIcon } from '@heroicons/react/24/outline';
+
+const navigation = {
+  main: [
+    { name: 'About', href: '#' },
+    { name: 'Documentation', href: 'https://github.com/czhu12/react-importer' },
+    { name: 'GitHub', href: 'https://github.com/czhu12/react-importer' },
+    {
+      name: 'Support',
+      href: 'https://stackoverflow.com/questions/tagged/react-importer',
+    }, // TODO: If we change the name to OneImport, we should change this
+  ],
+  social: [
+    {
+      name: 'Documentation', // TODO: Include a link here or remove
+      href: '#',
+      icon: (props: any) => <BookOpenIcon {...props} />,
+    },
+    {
+      name: 'GitHub',
+      href: '#',
+      icon: (props: any) => (
+        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
+          <path
+            fillRule="evenodd"
```

Comment:
Maybe we could have this and youtube icons as separate components in separate files?